### ISSUE TITLE
How are the train_metadata generated?

### ISSUE THREAD

**Issue by yxchng** [yifan123/flow_grpo#4]

**Comment by yifan123** [comment]
Training prompts are generated using scripts from [GenEval](https://github.com/djghosh13/geneval/blob/main/prompts/create_prompts.py). The test set is strictly deduplicated: prompts differing only in
object order (e.g., "a photo of A and B" vs. "a photo of B and A") are treated as identical and these variants are removed from the training set.
